subprojects/core-api/src/main/java/org/gradle/api/artifacts/dsl/DependencyCollector.java

```diff
@@ -16,6 +16,7 @@
 
 package org.gradle.api.artifacts.dsl;
 
+import com.h0tk3y.kotlin.staticObjectNotation.Restricted;
```

Comment:
Why is this not a Gradle package name? 

I think we should update the prototype to use a Gradle package before we start using it in the gradle codebase

Reply:
Good catch - I didn't notice the packages had changed after the merge to `master`.  I'll update this to use the new `org.gradle` ones.

Reply:
Hmmm, this isn't true.  The Restricted annotation hasn't moved to `org.gradle.internal.restricteddsl`.

Reply:
https://github.com/gradle/gradle/blob/74b5468e20a009b1bdc532fd5c88bb94248d8a6b/platforms/core-configuration/declarative-dsl-api/src/main/java/org/gradle/declarative/dsl/model/annotations/Restricted.java#L23

Not that one?